lib/src/main/kotlin/com/lemonappdev/konsist/api/provider/KoArgumentPropertiesProvider.kt

```diff
@@ -0,0 +1,17 @@
+package com.lemonappdev.konsist.api.provider
+
+/**
+ * An interface representing a Kotlin declaration that provides access to argument properties.
+ *
+ */
+interface KoArgumentPropertiesProvider : KoBaseProvider {
+    /**
+     * The argument name.
+     */
+    val argumentName: String?
```

Comment:
this should be just `name`, there is not need to define another property that holds